propagation/index.qmd

```diff
@@ -0,0 +1,131 @@
+
+Once your workflow is ready, you can create a UDP by parameterizing the process grap. [This notebook](https://documentation.dataspace.copernicus.eu/notebook-samples/openeo/UDP.html)
+demonstrates exactly how to do that.
+
+After your UDP is created, it can be registered in the APEx Algorithm Catalogue and associated with one or more backends capable of running it.
+As APEx continues to evolve, additional tools will become available that aid this process, including functionality to check if your service
+is still functional.
+
+For further guidance on creating and using UDPs, you can explore the following resources:
+
+- [APEx openEO process (UDP) writer guide](../udp_writer_guide.qmd#sec-udp-writing)
+- [APEx openEO process (UDP) integration guide](../udp_writer_guide.qmd#sec-udp-integration)
+- [UDP support in Python](https://open-eo.github.io/openeo-python-client/udp.html#user-defined-processes)
+- [Jupyter notebook example](https://github.com/Open-EO/openeo-community-examples/blob/main/python/Sentinel1_Stats/Sentinel1_Stats.ipynb)
+- [Official openEO specification](https://api.openeo.org/#tag/User-Defined-Processes)
+
+
+### Building an OGC application package based service
+
+Another option is to package your software using Docker containers. This approach is very generic and is particularly relevant
+if you have an existing piece of complex software that you want to make available as a service.
+
+[This tutorial](https://terradue.github.io/ogc-eo-application-package-hands-on/) explains more about this option.
+
+### What are the platforms to choose from?
+
+To offer your algorithm as an on-demand service, it must be available for execution on an APEx-compliant
+algorithm hosting platform. Currently, APEx supports the integration of services on platforms that adhere to the
+APEx [compliance guidelines](../interoperability/algohostingenv.md). It is important to note that this list of
+supported platforms is expected to grow as APEx evolves.
+
+For openEO UDPs:
+
+* [openEO platform](https://openeo.cloud)
+* [CDSE openEO federation](https://dataspace.copernicus.eu)
+
+For OGC Processes/Application Package:
+
+* [ELLIP](https://ellip.terradue.com)
+
+:::{.callout-tip}
+# Is Your Platform APEx-Compliant?
+If your platform meets the APEx [interoperability and compliance guidelines](../interoperability/algohostingenv.md),
+we encourage you to reach out to the APEx team to get it added to the list.
+:::
+
+### Onboarding your service on the ESA Network of Resources
+
+A key feature of the resulting on-demand services is that they can be onboarded in the ESA [Network of Resources (NoR)](https://nor-discover.org/).
+This integration enhances the service’s visibility and opens up a potential additional revenue stream for the project.
+
+An important step of the onboarding on the NoR is setting the price, expressed in euros per km². The final price of the
```

Comment:
not necessarily per km2 
it really depends on the function, e.g. it can be per file

Reply:
So can we list what options are 'allowed', or provide more accurate description of the intent?
- per km² of output data
- per input product 
- per output km² x time range x band count

I believe the main requirement is that it should be a 'predictable' measure, so not depending on cpu hours. Next to that, similar services should be comparable price-wise. So avoiding arbitrary differences is important.

Reply:
From a user point of view, doesn't it make sense to align and simplify (as much as possible) the final cost of a service? 

Based on previous experiences, we mainly see that users are having challenges with estimating the total cost for their purpose. Having a wide range of possible units could make this even more challenging, additionally, as users may have a difficult time translating their needs to certain units, such as the number of files or input products being processed.

Reply:
I agree with @JanssenBrm, so disregard my previous suggestion, or at least the one about 'per input product'.
For providers it's a relatively small thing to convert a price 'per file' to 'per km²', especially given that most 'files' have a similar size. When the file size is varying a lot, then the 'per file' pricing would also not make a lot of sense.